docs/user/azure/install.md

```diff
@@ -8,50 +8,6 @@ IPI install.
 The steps for performing an IPI-based install are outlined [here][cloud-install]. Following this guide you may begin at
 Step 3: Download the Installer.
 
-### Setup your Red Hat Enterprise Linux CoreOS images
```

Comment:
Just checking up on when these docs went obsolete, this section was automated in #1976.

Reply:
#2115 obsoleted this